### ISSUE TITLE
Why are subsciption videos different from those on Youtube?

### ISSUE THREAD

**Issue by czmiok** [FreeTubeApp/FreeTube#1197]
Why are there some missing videos in FreeTube in oppose to Youtube's site? 
I don't see some videos in FreeTube subscriptions tab that appear on YT subscriptions normally. Is there an other way this app manages those things. I've heard that YT hides videos sometimes, but I would expect the opposite from an app.
Yeah, I tried refreshing.

Disclaimer:
First time posting on GitHub and not a native speaker.

**Comment by GilgusMaximus** [comment]
Hi, could you give me an example of a channel? 
Personally I do not have any experience of missing videos or similar?

**Comment by czmiok** [comment]
I was about to list some channels but then I saw that even though I am subscribed to them on YT, I don't have a subscription on FreeTube. There was a problem during import. I've tried importing again and it did better but missed 3 subs. There was an error 'One or more subscriptions were unable to be imported' but I don't know if it was Youtube messing up an import file or FreeTube's problem. Nevertheless it works now.
